File: Java/Medium/2938.java
class Solution {
    public long minimumSteps(String s) {
        // is just bubble sort
        // count the number of black before that represents number of swaps

        long answer = 0;
        long black = 0;

        for (char c : s.toCharArray()) {
            if (c == '1') {// black
                black++;
            } else {// white
                answer += black;
            }
        }

        return answer;

    }
}

File: Python/Medium/2938.py
class Solution:
    def minimumSteps(self, s: str) -> int:
        # is just bubble sort
        # count number of blacks before white represents number of swaps needed
        answer = 0
        black = 0

        for c in s:
            if c == "1":  # black
                black += 1
            else:  # white
                answer += black

        return answer
